microsoft/testsuites/network/common.py

```diff
@@ -161,24 +115,30 @@ def sriov_vf_connection_test(
             f" machine {source_node.name}, please check network setting of "
             f"machine {dest_node.name}."
         ).is_not_empty()
-        desc_nic_info = vm_nics[dest_node.name][matched_dest_nic_name]
+        dest_nic_info = vm_nics[dest_node.name][matched_dest_nic_name]
         dest_ip = vm_nics[dest_node.name][matched_dest_nic_name].ip_addr
         source_ip = source_nic_info.ip_addr
-        source_synthetic_nic = source_nic_info.upper
-        dest_synthetic_nic = desc_nic_info.upper
-        source_nic = source_vf_nic = source_nic_info.lower
-        dest_nic = dest_vf_nic = desc_nic_info.lower
+        source_synthetic_nic = source_nic_info.name
+        dest_synthetic_nic = dest_nic_info.name
+        source_nic = source_pci_nic = source_nic_info.pci_device_name
+        dest_nic = dest_pci_nic = dest_nic_info.pci_device_name
 
-        if remove_module or turn_off_vf:
+        if remove_module or turn_off_pci_nic:
             source_nic = source_synthetic_nic
             dest_nic = dest_synthetic_nic
-        if turn_off_vf:
-            source_node.execute(f"ip link set dev {source_vf_nic} down", sudo=True)
-            dest_node.execute(f"ip link set dev {dest_vf_nic} down", sudo=True)
+
+        if turn_off_pci_nic:
```

Comment:
The original logic to be true for all pci_devices. With the condition `source_nic_info.is_pci_device and not source_nic_info.is_pci_module_enabled`, most pci devices skip the logic.

Reply:
when no lower nic, we can't set down the nic

Reply:
the variable name is confusing, how about `turn_off_lower`?

Reply:
updated